components/Navbar.vue

```diff
@@ -399,8 +399,12 @@ export default class NavbarMenu extends mixins(PrefixMixin) {
     width: min-content;
     min-width: 140px;
     margin: 0 1rem;
-    .icon {
-      color: $placeholder-color !important;
+    .gallery-search {
+      .control {
+        .icon {
+          color: $placeholder-color !important;
+        }
+      }
```

Comment:
I don't understand why you need to rewrite this rule?

**on beta already work**
![Capture d’écran 2022-08-22 à 9 48 03 AM](https://user-images.githubusercontent.com/9987732/185867921-f332da1e-df6a-4350-8738-36328fd8399b.png)

**on preview** (rewriting rule)
![Capture d’écran 2022-08-22 à 9 48 16 AM](https://user-images.githubusercontent.com/9987732/185869875-93c8326c-c192-4f9d-81d3-5f8df6ef4982.png)

personnal preference: no need to write multiple line only add `.gallery-search .control .icon`